controllers/upgrade.go

```diff
@@ -262,10 +267,6 @@ func (r *BookkeeperClusterReconciler) syncBookkeeperVersion(bk *bookkeeperv1alph
 	}
 
 	// If all replicas are ready, upgrade an old pod
```

Comment:
Please move the comment `// If all replicas are ready, upgrade an old pod` to line 249